go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2804,6 +2816,13 @@ func (e *Executor) gcArtifacts(ctx context.Context) error {
 				return err
 			}
 		}
+
+		// while the next fnuction only applies to 'online' strategy ALTER and REVERT, there is no
```

Comment:
If you decide to address the "extra braces" comment,  you might want to also fix the typo in this comment.

Reply:
fixed typo